pkg/xds/server/callbacks/dataplane_status_sink_test.go

```diff
@@ -60,6 +62,11 @@ var _ = Describe("DataplaneInsightSink", func() {
 			ticker := &time.Ticker{
 				C: ticks,
 			}
+			metrics, err := core_metrics.NewMetrics("")
```

Comment:
do we trigger any of the new paths in this tests?

https://github.com/kumahq/kuma/blob/8e9d4c16461f57e8da6c008f4aa6c1ce2bb66d56/pkg/xds/server/callbacks/dataplane_status_sink.go#L135-L150